dbreps2/src/general.rs

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2021 Kunal Mehta <legoktm@debian.org>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+mod excessiveips;
+mod indeffullredirects;
+mod oldeditors;
+mod pollcats;
+mod uncatcats;
+
+pub use {
+    excessiveips::ExcessiveIps, indeffullredirects::IndefFullRedirects,
+    oldeditors::OldEditors, pollcats::Pollcats, uncatcats::UncatCats,
```

Comment:
suggest using one line per entry, to make the diffs clearer if more are added or one is removed

Reply:
This is just how `rustfmt` formats it. It is a bit annoying but I really just appreciate that it formats everything for us and I don't need to worry about it.